Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -1324,17 +1324,17 @@ CSSTransformStyleOptimized3DEnabled:
 
 CSSTypedOMColorEnabled:
   type: bool
-  status: stable
+  status: testable
```

Comment:
We don't want this in Preview yet? I suppose its too incomplete?

Reply:
The implementation is incomplete. This part of the spec is still unstable.